packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -958,6 +976,7 @@ Future<T?> showMenu<T>({
   required BuildContext context,
   required RelativeRect position,
   required List<PopupMenuEntry<T>> items,
+  List<GlobalKey>? itemKeys,
```

Comment:
What's the use case for exposing these in the API?

If we do expose them, please cover them in the dart docs above.

Reply:
Oh sorry, I think we don't have to expose this. I used it to do some experiments but forgot to clean it up! Updating.